java/openjdk-java/portfile

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                openjdk-java
+version             17
+categories          java
+platforms           darwin
+description         an Open Source, Open Innovation, Open Standards implementation of the Java Platform, Standard Edition, and related projects.
+homepage            openjdk.java.net
+license             GPL-2+CE
```

Comment:
This is not a recognized license name. What's the "CE" part?

Reply:
CE is classpath exception see (https://openjdk.java.net/legal/gplv2+ce.html)

Reply:
I can remove CE part and keep only GPL-2 as CE is just linking exception

Reply:
removed ce part